README.md

```diff
@@ -20,6 +20,10 @@ Run ```npm start```.
 * ```HTML_AUTHOR```: The HTML author of the app.
 * ```HTML_DESCRIPTION```: The HTML description of the app.
 * ```NAME_PATTERN```: The pattern the names have to match to join a session. Default is ```.*```
+* ```BASE_URL```: The base URL containing request protocol (http | https), domain and optionally port, eg. ```https://example.com:8443```
```

Comment:
This makes `OAUTH_REDIRECT_BASE` obsolete and it should be removed below in [line 34](https://github.com/BDSU/meeting-moderation-cards/pull/9/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R34)